doc/api/assert.md

```diff
@@ -9,6 +9,8 @@ test invariants.
 <!-- YAML
 added: v0.5.9
 -->
+* `value` {Boolean|Number}
```

Comment:
Well, technically any value (not just booleans and numbers) would work. I'd be fine with either `any` or `Boolean`. since the concepts of truthy-ness and falsy-ness are defined by `Boolean(val)`.